src/PerformanceTests/NServiceBus7/App.config

```diff
@@ -106,24 +89,4 @@
     </assemblyBinding>
     <gcServer enabled="false" />
   </runtime>
-  <system.net>
```

Comment:
Do not remove `system.net`.

Reply:
O wait, I see, you've swapped it for code API.

Reply:
dotnet core's configurationmanager package isn't able to load system.net sections and will throw.